app/views/catalog/_document_split.html.erb

```diff
@@ -1,6 +1,8 @@
 <div class='row'>
   <div id="documents" class="documents-list col-md-6">
-    <%= render documents, :as => :document %>
+    <% documents.each_with_index do |document, index| %>
+      <%= render Geoblacklight::SearchResultComponent.new(document: document, document_counter: index) %>
```

Comment:
Could you do with_collection instead of a loop? I am not 100% sure this work

<%= render Geoblacklight::SearchResultComponent.with_collection(documents.each_with_index) %>

Reply:
Somehow it automagically works to do 
`<%= render Geoblacklight::SearchResultComponent.with_collection(documents) %>
`
-- and it knows what to pass for the `document:` and `index:` params in the initializer in the VC Class.